ui/src/app/components/session-listing/session-listing.component.html

```diff
@@ -1,26 +1,100 @@
 <div class="sessions-wrapper">
   <h3 class="conv-history">Conversion history</h3>
-  <div class="summary">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</div>
+  <div class="summary">
+    Choose a session to resume an existing migration session, or download the session file.
+  </div>
   <br />
   <div class="session-container mat-elevation-z3" *ngIf="dataSource.length > 0">
-    <table mat-table [dataSource]="dataSource">
+    <table mat-table [dataSource]="filteredDataSource">
       <ng-container matColumnDef="SessionName">
-        <th mat-header-cell *matHeaderCellDef>Session Name</th>
+        <th mat-header-cell *matHeaderCellDef>
+          <div class="table-header-container">
+            <span>Session Name</span>
+            <button mat-icon-button (click)="toggleFilterDisplay('sessionName')">
+              <mat-icon *ngIf="!displayFilter.sessionName">arrow_drop_down</mat-icon>
```

Comment:
The arrow icons denote sorting mostly , it would be better to replace these with filter icons

Reply:
Updated